serialization/avro-serializer/src/main/java/hazelcast/avro/Person.java

```diff
@@ -0,0 +1,197 @@
+/**
+ * Autogenerated by Avro
+ *
+ * DO NOT EDIT DIRECTLY
+ */
+package hazelcast.avro;
+
+import org.apache.avro.specific.SpecificData;
+
+@SuppressWarnings("all")
+@org.apache.avro.specific.AvroGenerated
+public class Person extends org.apache.avro.specific.SpecificRecordBase implements org.apache.avro.specific.SpecificRecord {
```

Comment:
maybe instead of committing generated file, we can update README to show how to generate it... That can make easier for people who want to play with the code... That is just a suggestion though :)